contracts/colony/Colony.sol

```diff
@@ -60,6 +63,9 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
 
     require(sig != APPROVE_SIG, "colony-cannot-call-erc20-approve");
     require(sig != TRANSFER_SIG, "colony-cannot-call-erc20-transfer");
+    require(sig != TRANSFER_FROM_SIG, "colony-cannot-call-erc20-transfer-from");
+    require(sig != BURN_SIG, "colony-cannot-call-burn");
```

Comment:
Shouldn't these messages include the `erc20-` component?

Reply:
The [ERC20 Spec](https://eips.ethereum.org/EIPS/eip-20#methods) does not include the `burn` functions.